crates/ruma-client-api/src/r0/backup.rs

```diff
@@ -81,3 +100,10 @@ pub struct SessionData {
     /// First 8 bytes of MAC key, encoded in base64.
     pub mac: String,
 }
+
+impl SessionData {
+    /// Create a `SessionData` struct.
+    pub fn new(ephemeral: String, ciphertext: String, mac: String) -> Self {
```

Comment:
Since these are all `String`s, I think an `Init` struct would be more appropriate.